### PR TITLE
Feature/oleh branch

### PR DESCRIPTION
**Pull request by glckfndr**

### CODE REVIEW

File: students/oleh.html [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello!
```
Comment (ViraKryvets) [inline students/oleh.html:1]: ok

**Review by ViraKryvets — APPROVED** [review]

**Comment by glckfndr** [comment]
The first change

File: index.html [hunk 2]
```diff
@@ -0,0 +1 @@
+First change!
```
Comment (AndriyAndriyovuch) [inline index.html:1]: nice

**Review by AndriyAndriyovuch — APPROVED** [review]

**Review by lbeketl — APPROVED** [review]
ok

**Review by kos-nastiia — APPROVED** [review]

**Pull request merged**